routers/api/v1/api.go

```diff
@@ -590,6 +590,7 @@ func Routes() *web.Route {
 			})
 		}
 		m.Get("/version", misc.Version)
+		m.Get("/nodeinfo", misc.NodeInfo)
```

Comment:
Not familiar with [forgefed specs](https://notabug.org/peers/forgefed) at all (yet!), so please ignore if it's a stupid remark:
Wouldn' it make sense for this to go under `/.well-known/nodeinfo` or something?
And/or maybe put all activitypub related APIs under it's own namespace / router (so the feature is easy to opt in/out)?

Reply:
Ooh, now I see it down below :D That looks like a good scheme :>